docs/outlook/use-sso-in-event-based-activation.md

```diff
@@ -1,16 +1,16 @@
 ---
```

Comment:
Since this applies to WXP add-ins as well, consider moving this article to the `develop` folder. If you make this change, you'll also need to update the TOC.